src/net_processing.cpp

```diff
@@ -808,13 +812,15 @@ class PeerManagerImpl final : public PeerManager
     std::unique_ptr<TxReconciliationTracker> m_txreconciliation;
     const std::unique_ptr<CDeterministicMNManager>& m_dmnman;
     const std::unique_ptr<ActiveContext>& m_active_ctx;
-    /** Pointer to this node's CJWalletManager. May be nullptr - check existence before dereferencing. */
-    CJWalletManager* const m_cj_walletman;
     const std::unique_ptr<LLMQContext>& m_llmq_ctx;
+    [[maybe_unused]] const std::unique_ptr<llmq::ObserverContext>& m_observer_ctx;
     CMasternodeMetaMan& m_mn_metaman;
     CMasternodeSync& m_mn_sync;
     CGovernanceManager& m_govman;
     CSporkManager& m_sporkman;
+    /** Pointer to this node's CJWalletManager. May be nullptr - check existence before dereferencing. */
```

Comment:
nit: seems irrelevant to re-order `m_cj_walletman` for ` refactor: create stub class for watch-only mode context `

Downsides: increase diff size; increase probability of conflicts with other PRs; pollute `git blame` output